lekko_client/__init__.py

```diff
@@ -48,7 +48,7 @@ class APIConfig(Config):
 
 @dataclass(kw_only=True)
 class CachedServerConfig(Config):
-    pass
+    update_interval_ms: int = 1000
```

Comment:
This feels aggressive (sidecar defaults to 15s)

Reply:
This is what we already use in python. Agree that we should unify it and 1s is probably too short. Looks like it's 15s in go as well: https://github.com/lekkodev/go-sdk/blob/f072eb8bf64e727859a5bb0f3d2cddc0b4a4fe7e/client/cached_provider.go#L32
Will change it to 15s.